src/bacnet/basic/object/ai.c

```diff
@@ -45,6 +45,7 @@ static const int Properties_Optional[] = {
     PROP_TIME_DELAY, PROP_NOTIFICATION_CLASS, PROP_HIGH_LIMIT,
     PROP_LOW_LIMIT, PROP_DEADBAND, PROP_LIMIT_ENABLE, PROP_EVENT_ENABLE,
     PROP_ACKED_TRANSITIONS, PROP_NOTIFY_TYPE, PROP_EVENT_TIME_STAMPS,
+    PROP_EVENT_DETECTION_ENABLE,
```

Comment:
5 These properties shall be present only if the object supports intrinsic reporting. ;)